waiter/src/waiter/request_log.clj

```diff
@@ -93,7 +93,11 @@
 (defn log-request!
   "Log a request"
   [request response]
-  (log (merge (request->context request) (response->context response))))
+  (let [redacted-request-fields-string (get-in response [:descriptor :service-description "env" "REDACTED_REQUEST_FIELDS"])]
+    (log (apply dissoc
```

Comment:
It's confusing having this much logic in the `log` statement. Let's lift the redacted request into a new `let` binding.